### DISCUSSION TITLE
How to set postgres schema for model

### DISCUSSION THREAD

**Discussion by rorik302** [collerek/ormar#459] [Q&A]
Hi, 
I am trying to build multitenant app, but I'm stuck with a problem how to set schema for a model. I make two alembic apps for public and for tenants, but alembic don't know for what database schema make revision or migrate because models schema is None. And if solution exists, will `ForeignKey` work between public and tenants schemas?

**Comment by collerek** [comment]
Right now it's not possible - ormar uses no schema as of now (so a default one) for all tables.

Already reported in #287
